openfl/interface/workspace.py

```diff
@@ -387,13 +385,13 @@ def export_() -> str:
     "--save",
     is_flag=True,
     default=False,
-    help="Export the docker image as <workspace_name>.tar file.",
+    help="Export the docker image as ``<workspace_name>.tar`` file.",
```

Comment:
Add `show_default=True`?

Reply:
Click does not print default values when flags are boolean and `False` as default